tls/go-server/app/env.sh

```diff
@@ -0,0 +1,2 @@
+export CONF_PROVIDER_FILE_PATH="../profiles/dev/server.yml"
```

Comment:
avoid use shell script to start. try if build/makefile work or not. pls. refer to HOWTO.md